src/lib/holocene/button.svelte

```diff
@@ -1,222 +1,139 @@
-<script lang="ts" context="module">
-  type ButtonVariant =
-    | 'primary'
-    | 'secondary'
-    | 'search'
-    | 'destructive'
-    | 'login'
-    | 'ghost'
-    | 'link'
-    | 'menu';
+<script lang="ts">
```

Comment:
Also, in no particular order r.e. some other button related components...

Is this the state we want for this icon button? Hover bleeds over header a bit (and it's also on a black background already).
![Screenshot 2023-08-29 at 4 11 02 PM](https://github.com/temporalio/ui/assets/15069288/a87548a0-289d-46f4-9533-67c6db5c92af)



For the copyable icon button,
![Screenshot 2023-08-29 at 3 51 53 PM](https://github.com/temporalio/ui/assets/15069288/64b0bab8-4b2d-4853-b3f6-baa6b8c0ef94)

wondering if we want it to be closer to what we have in the `CodeBlock` component?
![Screenshot 2023-08-29 at 3 51 39 PM](https://github.com/temporalio/ui/assets/15069288/26749ffe-fa94-40eb-a206-fae43e0ab1ed)
And also, if we want to change the fill color of the icon on hover when its a link?
![Screenshot 2023-08-29 at 3 57 54 PM](https://github.com/temporalio/ui/assets/15069288/68863106-0a4c-4f86-a9ff-8b57ea2ecdb1)

For the `Accordion` component, wondering if this is the style we want for the chevron icon (even when disabled)? Or do we want to make this more similar to the `Button` styling?
![Screenshot 2023-08-29 at 4 15 29 PM](https://github.com/temporalio/ui/assets/15069288/0b3348e6-d3d3-412b-afe2-f90c10354576)

Reply:
Good call, for Icon button I changed the hover/focus state to our blue->purple gradient, so it should be consistent now.